phpBB/styles/prosilver/template/posting_pm_header.html

```diff
@@ -1,83 +1,110 @@
+											<input type="submit" name="remove_{{ to_recipient.TYPE }}[{{ to_recipient.UG_ID }}]" value="x" class="button1 button button-form-bold">
+										{% endif %}
+										{% if to_recipient.IS_GROUP %}
+											<a href="{{ to_recipient.U_VIEW }}" style="color: {{ to_recipient.COLOUR }}"><strong>{{ to_recipient.NAME }}</strong></a>
+										{% else %}
+											{{ to_recipient.NAME_FULL }}
+										{% endif %}
+									</li>
+								{% endfor %}
+							</ul>
+						</dd>
+					</dl>
+				{% endif %}
+			</div>
+			{% if .bcc_recipient %}
+				<div class="column2">
+					<dl>
+						<dt><label>{{ lang('BCC') ~ lang('COLON') }}</label></dt>
+						<dd class="recipients">
+							<ul class="recipients">
+								{% for bcc_recipient in .bcc_recipient %}
+									<li>
+										{% if not S_EDIT_POST %}
+											<input type="submit" name="remove_{{ bcc_recipient.TYPE }}[{{ bcc_recipient.UG_ID }}]" value="x" class="button1 button button-form-bold">
+										{% endif %}
+										{% if bcc_recipient.IS_GROUP %}
+											<a href="{{ bcc_recipient.U_VIEW }}" style="color: {{ bcc_recipient.COLOUR }}"><strong>{{ bcc_recipient.NAME }}</strong></a>
+										{% else %}
+											{{ bcc_recipient.NAME_FULL }}
+										{% endif %}
+									</li>
+								{% endfor %}
+							</ul>
+						</dd>
+					</dl>
+				</div>
+			{% endif %}
+		{% else %}
+			<div class="column1">
 				<dl>
-					<dt><label>{L_BCC}{L_COLON}</label></dt>
-					<dd class="recipients">
-					<ul class="recipients">
-						<!-- BEGIN bcc_recipient -->
-						<li>
-							<!-- IF not S_EDIT_POST --><input type="submit" name="remove_{bcc_recipient.TYPE}[{bcc_recipient.UG_ID}]" value="x" class="button1 button button-form-bold" /><!-- ENDIF -->
-							<!-- IF bcc_recipient.IS_GROUP --><a href="{bcc_recipient.U_VIEW}" style="color: {{ bcc_recipient.COLOUR }}"><strong>{bcc_recipient.NAME}</strong></a><!-- ELSE -->{bcc_recipient.NAME_FULL}<!-- ENDIF -->
-						</li>
-						<!-- END bcc_recipient -->
-					</ul>
-					</dd>
+					<dt>
+						<label for="username_list">{{ lang('TO_ADD') ~ lang('COLON') }}</label>
+						{% if not S_EDIT_POST %}
+							<br><span><a href="{{ U_FIND_USERNAME }}" onclick="find_username(this.href); return false">{{ lang('FIND_USERNAME') }}</a></span>
+						{% endif %}
+					</dt>
+					{% if not S_EDIT_POST %}
+						<dd>
+							<input class="inputbox" type="text" name="username_list" id="username_list" size="20" value=""> <input type="submit" name="add_to" value="{{ lang('ADD') }}" class="button1 button button-form-bold">
+						</dd>
+					{% endif %}
+					{% if .to_recipient %}
+						<dd class="recipients">
+							<ul class="recipients">
+								{% for to_recipient in .to_recipient %}
+									<li>
+										{% if to_recipient.IS_GROUP %}
+											<a href="{{ to_recipient.U_VIEW }}"><strong>{{ to_recipient.NAME }}</strong></a>
+										{% else %}
+											{{ to_recipient.NAME_FULL }}
+										{% endif %}&nbsp;
```

Comment:
What is the space doing here exactly? And would it not make more sense in the if below?

Reply:
Might be a merge conflict resolution issue, will check it

Reply:
Exists in the previous code: https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/template/posting_pm_header.html#L72

Reply:
But I agree, makes more sense in the if below.